Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
I wouldn't thane an issue with that. Thanks.

As a side note, "to condescend", "to be" , "to do" or any other verb is just a way to refer to its infinitive form. Therefore, it's the right way to translate "condescender", "ser", "hacer", etc.

Think about "I like to play". It translates to Spanish as "me gusta jugar", and NEVER as "me gusta a jugar".